starter-code/scripts/controllers/articleController.js

```diff
@@ -13,16 +13,19 @@
     }
   };
 
-  // COMMENT: What does this method do?  What is it's execution path?
+  // COMMENTed: What does this method do?  What is it's execution path?
+  // This method redirects the page to load the remaining portion of the article upon clicking on read-on. Its execution path would be upon click of read on it would look through all instances of the artciles, locate the article ID you selected, findwhere the article ID is the same as the selected article and then calls the next function articleControll.index which passes the article content into the articleView.index as an arguement, if an article ID was not found it will reload the home page.
   articleController.loadById = function(ctx, next) {
+    console.log(ctx);
     var articleData = function(article) {
       ctx.articles = article;
       next();
     };
     Article.findWhere('id', ctx.params.id, articleData);
   };
 
-  // COMMENT: What does this method do?  What is it's execution path?
+  // COMMENTed: What does this method do?  What is it's execution path?
+  //based on the seleted option from the navigation menu it finds all articles with the same author value and displays those articles. Upon selection loadByAuthor is invoked and then it runs articleController.index when renders the correctly selected articles to the view.
```

Comment:
Correct! This method calls Article.findWhere, which queries the database for articles with an author matching the ctx.params.authorName property (passed via the route being hit in articleView.handleFilters).  It passes in authorData, defined above, as a callback to Article.findWhere, so after the query, an articles property is attached to our context object containing all of the articles that match our query. Then, the next function in the middleware chain is called (articleController.index).